megamek/data/rat/Turret RATs/Actual Turrets/InDirectFire Turrets 3050.txt

```diff
@@ -0,0 +1,13 @@
+InDirectFire Turrets 3039
+
+#Missile Turrets
+Light Missile Turret (3039), 50
+Medium Missile Turret (3039), 25
+Heavy Missile Turret (3039), 15
+Assault Missile Turret (3039), 5
+
+#Bombard Turrets
+Light Bombard Turrets (3050), 50
```

Comment:
Should this be "Light Bombard Turret", or a reference like `@Light Bombard Turrets`?